src/pages/Join/Join.js

```diff
@@ -1,11 +1,149 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import '../Join/Join.scss';
 
 function Join() {
-    return (
-        <div>
-            
+  const navigate = useNavigate();
+
+  const [inputValue, setInputValue] = useState({
+    email: '',
+    password: '',
+    name: '',
+    address: '',
+  });
+
+  const inputChange = e => {
+    setInputValue({ ...inputValue, [e.target.name]: e.target.value });
+  };
+
+  const { email, password, name, address } = inputValue;
+
+  const [isCheckEmail, setIsCheckEmail] = useState(false);
+
+  const checkDuplicate = e => {
+    e.preventDefault();
+
+    fetch('http://10.58.2.130:3001/users/signup/user', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json;charset=utf-8',
+      },
+      body: JSON.stringify({
+        email: email,
+      }),
+    })
+      .then(response => {
+        if (response.ok === true) {
+          return response.json();
+        }
+        throw new Error('에러 발생!');
+      })
+      .catch(error => {
+        alert(error);
+      })
+      .then(data => {
+        if (data.message === 'EXCESS_SUCCESS') {
+          alert('사용 가능한 이메일입니다.');
+          setIsCheckEmail(true);
+        } else if (data.message === 'KEY_ALREADY_EXISTS') {
+          alert('이미 사용중인 이메일입니다.');
+          setIsCheckEmail(false);
+        }
+      });
+  };
+
+  const submitForm = e => {
+    e.preventDefault();
+
+    if (isCheckEmail) {
+      fetch('http://10.58.2.130:3001/users/signup', {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json;charset=utf-8',
+        },
+        body: JSON.stringify({
+          email: email,
+          name: name,
+          password: password,
+          address: address,
```

Comment:
key와 value가 같다면 문법적으로 다음과 같이 쓸 수도 있습니다. 참고만 해주세요!
```suggestion
          email,
          name,
          password,
          address,
```